src/poprox_recommender/handler.py

```diff
@@ -3,8 +3,9 @@
 
 from poprox_concepts import ArticleSet
 from poprox_concepts.api.recommendations import RecommendationRequest, RecommendationResponse
+from poprox_recommender.components.diversifiers.locality_calibration import user_interest_generate
```

Comment:
The handler shouldn't know anything about individual recommenders, so importing code from this recommender-specific component is a clue that the code below is probably in the wrong place. Could these changes live somewhere in the `locality_calibration` module?

Reply:
Moved to` locality_calibration `module

Reply:
Just to confirm, could we modify the code in the handler.py in our forked repository?